scriptworker/cot/verify.py

```diff
@@ -1393,14 +1392,13 @@ def get_firefox_source_url(obj):
     repos = {}
     restricted_privs = None
     rules = {}
-    cot_product = chain.context.config['cot_product']
     for my_key, config_key in {
         'scopes': 'cot_restricted_scopes',
         'trees': 'cot_restricted_trees'
     }.items():
-        rules[my_key] = chain.context.config[config_key].get(cot_product)
+        rules[my_key] = chain.context.config[config_key]
         if not isinstance(rules[my_key], (dict, frozendict)):
-            raise_on_errors(["{} invalid for {}: {}!".format(config_key, cot_product, rules[my_key])])
```

Comment:
Thinking more about this check, now that the config type-checking looks through keyed config entries, I don't think this is needed anymore.